src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -77,6 +77,15 @@ public void enterXMode() {
     drive.lockPose();
   }
 
+  /**
+   * Set the heading correction capabilitys of YAGSL.
+   *
```

Comment:
I'd add another paragraph here that says something like "Should only be enabled when heading correction capabilities are in use"